src/user-event/type/type.ts

```diff
@@ -69,7 +70,9 @@ export async function type(
 
   dispatchEvent(element, 'endEditing', EventBuilder.TextInput.endEditing(finalText));
 
-  dispatchEvent(element, 'blur', EventBuilder.Common.blur());
+  if (!options?.skipBlur) {
```

Comment:
skip blur option should disable both endEditing and blur together, as in in native platforms (iOS, android) they always happen together.

Reply:
Made the changes on the latest commit.